zookeeper/versions.py

```diff
@@ -1,14 +1,8 @@
 versions = [
-    {
-        "product": "3.8.4",
-        "java-base": "11",
-        "java-devel": "11",
-        "jmx_exporter": "1.0.1",
-    },
     {
         "product": "3.9.2",
-        "java-base": "11",
-        "java-devel": "11",
+        "java-base": "11",  # Builds fail on Java 17, so it can stay as is.
+        "java-devel": "11",  # Builds fail on Java 17, so it can stay as is.
```

Comment:
Should we put the "error" message or at least the package that failed in the comment for future ref?

Reply:
The error wasn't helpful at all, but I can mention the package.

Reply:
Fixed in b8487a3